dat/training.csv

```diff
@@ -0,0 +1,69 @@
+103079329921,114689,103079215232,2

```

Comment:
I think we can save some space by only saving the player1 bb and the player2 bb. The occupied bb we can simply calculate later with p1bb | p2bb